tasks/process-spectrum.js

```diff
@@ -146,698 +129,573 @@ function conditionSelector(newSelector) {
         }
         return acc;
     }, []);
-    return selector;
 }
 
-async function processComponent(componentPath) {
-    const { default: config } = await import(
-        path.join(componentPath, 'spectrum-config.js')
-    );
-    /**
-     * @type { import('./spectrum-css-converter').SpectrumCSSConverter}
-     */
-    const { conversions } = config;
+/**
+ * Process the selectors array and return the processed selectors
+ * @param {import('./spectrum-css-converter').HostSelectorComponent} selector
+ * @param {import('./spectrum-css-converter').ComponentConversion[]} components
```

Comment:
there is no `ComponentConversion` type exported in this repo

Reply:
Alrighty, no worries, the types aren't particularly detailed in the local file so for now I'll just set them as any since they weren't documented at all before. We can always add more specific types later.